internal/capability/operand_cleanup.go

```diff
@@ -0,0 +1,67 @@
+package capability
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/opdev/opcap/internal/operator"
+
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// OperandCleanup removes the operand from the OCP cluster in the ca.namespace
+func (ca *capAudit) OperandCleanUp() error {
+	logger.Debugw("cleaningUp operand for operator", "package", ca.subscription.Package, "channel", ca.subscription.Channel, "installmode", ca.subscription.InstallModeType)
+
+	if len(ca.customResources) > 0 {
+		obj := &unstructured.Unstructured{Object: ca.customResources[0]}
```

Comment:
I'm a bit concerned about this ca.customResources[0]. This is not a change request but something we should talk about. How to bring the ALMExamples in and also to report that some operators have more than one and most likely examples to different APIs, meaning different CRDs that are deployed with the operator.

Reply:
Just adding the Issue #131

Reply:
I think we could loop over all customResources and then install each one of them as part of enhancement.

Reply:
Agreed. Let it as it is for now.